bullet_train-themes-light/lib/tasks/application.rb

```diff
@@ -3,6 +3,14 @@
 module BulletTrain
   module Themes
     module Application
+      def self.eject_theme_main_css(theme_name)
+        theme_base_path = `bundle show --paths bullet_train-themes-#{theme_name}`.chomp
+
+        puts "Ejecting app/assets/stylesheets/#{theme_name}.tailwind.css."
+        `mkdir -p #{Rails.root}/app/assets/stylesheets`
```

Comment:
If we use `Rails.root.join`, we get back a `Pathname` that I think we can call `mkpath` on and get the same effect:

```suggestion
        Rails.root.join("app/assets/stylesheets").mkpath
```